cugraph/README.md

```diff
@@ -0,0 +1,73 @@
+
+
+# cuGraph Notebooks
+
+![GraphAnalyticsFigure](img/GraphAnalyticsFigure.jpg)
+
+This repository contains a collection of Juyper Notebooks that outline how to run various cuGraph analytics.   The notebooks do not address a complete data science problem.  The notebooks are simply examples of how to run the graph analytics.  Manipulation of the data before or after the graph analytic is not covered here.   Extended, more problem focused, notebooks are being created and available https://github.com/rapidsai/notebooks-extended
```

Comment:
typo: Juyper->Juypter

Reply:
I blame the auto-correct spell checker :-)